lib/autoyast.pm

```diff
@@ -720,6 +721,23 @@ sub expand_variables {
     return $profile;
 }
 
+=head2 adjust_user_password
+
+ adjust_user_password($profile);
+
+ Use default one "$testapi::password" if "PASSWORD" is not defined
```

Comment:
This sentence is not clear, PASSWORD is just a placeholder, not an openQA variable, it is replaced unconditionally wherever it finds that string to replace, "PASSWORD" is never defined. Could you please improve it?